src/action_types/channels.ts

```diff
@@ -1,49 +1,35 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-// @flow
-
 import keyMirror from 'utils/key_mirror';
-
 export default keyMirror({
     CHANNEL_REQUEST: null,
     CHANNEL_SUCCESS: null,
     CHANNEL_FAILURE: null,
-
```

Comment:
The whitespace removed in the action type files kind of bloats the PR, and arguably makes the code harder to read. Do you think the blank lines served a good purpose?

0/5 We leave the newlines in the code

Reply:
This is a result of automatic conversion. I've tried to fix most of the formatting issues, but some are still lingering, not sure how to auto-fix this stuff.